qemu_test.go

```diff
@@ -871,22 +918,29 @@ func TestFullUEFISpiceMachineCommand(t *testing.T) {
 }
 
 func TestFullUEFITPMMachineCommand(t *testing.T) {
-	c := fullVMConfig()
-
-	u := UEFIFirmwareDevice{
-		Code: "/usr/share/OVMF/OVMF_CODE.fd",
-		Vars: "uefi_nvram.fd",
-	}
+	var c *Config
+	u := UEFIFirmwareDevice{Code: "", Vars: "uefi-nvram.fd",}
+	expected := ""
+	switch runtime.GOARCH {
+	case "aarch64", "arm64":
+		c = fullVMConfigAarch64()
+		u.Code = "/usr/share/AAVMF/AAVMF_CODE.ms.fd"
+		expected = fullUefiTPMAarch64VM
+	case "x86_64", "amd64":
+		c = fullVMConfig()
+		u.Code = "/usr/share/OVMF/OVMF_CODE.fd"
+		expected = fullUefiVMTPM
+	}
+	
 	c.UEFIFirmwareDevices = append(c.UEFIFirmwareDevices, u)
 
 	c.TPM = TPMDevice{
 		ID:     "tpm0",
 		Driver: TPMTISDevice,
-		Path:   "tpm.socket",
+		Path:   "tpm.sock",
```

Comment:
Let's skip this sort of change and just update the expected command line to match the existing constructed devices.